Automattic/woocommerce-payments
# Replace the "colors" dependency with chalk

Fixes #3961

#### Changes proposed in this Pull Request

Replace the "colors" dependency with [chalk](https://github.com/chalk/chalk). We only use it to style the output of a few messages on the `release` script, so this change has no risk. I was tempted to just drop the dependency and let the success & error messages at the end of the release script be boring white-on-black, but whatever, we're already pulling the `chalk` dependency transitively.

#### Testing instructions

- Run `npm run build:release`.
- Notice that the script doesn't crash.
- Notice that the script prints a green message at the end.
